src/club-description.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en-CA
+title: Guelph Coding Community
+subtitle: Club description
+date: "Last updated: September 2019"
+
+documentclass: scrartcl
+numbersections: false
+geometry:
+ - margin=1in
+
+urlcolor: blue
+fontsize: 11pt
+
+header-includes:
+ - \usepackage{enumitem}
+ - \usepackage{amsfonts}
+ - \setlist[itemize,1]{label=$\bullet$}
+ - \setlist[itemize,2]{label=$\circ$}
+...
+
+The Guelph Coding Community (GCC) is a student organization focused on providing students with valuable, hands-on, extracurricular tech skills which cannot normally be acquired through coursework.
```

Comment:
Rephrasing:
"... focused on valuable, hands-on, technical skills, which are not normally presented in course material."

Reply:
Or just the removal of the word "extracurricular". Seems superfluous to me.